klaviyo/api_helper.py

```diff
@@ -200,7 +236,35 @@ def _public_request(self, path, querystring):
         url = '{}/{}?{}'.format(self.api_server, path, querystring)
         return self._request(self.HTTP_GET, url, request_type=self.PUBLIC)
 
-    def _request(self, method, url, params=None, data=None, request_type=PRIVATE):
+    def _public_post_request(self, method=HTTP_GET, params=None, path=None, is_test=False):
```

Comment:
It seems like params kwarg should be an empty dict?
```suggestion
    def _public_post_request(self, method=HTTP_GET, params={}, path=None, is_test=False):
```

Reply:
i also don't see path inside the docstring

Reply:
I'd also like to call it resource rather than path, path to me seems like it should be a url path, which in this case it's more of the endpoint (resource) we'd call

Reply:
`None` is actually the proper way to define the kwarg here. If you modify the kwarg in the method the value will persist into subsequent calls. It's not relevant here because we don't modify but I'll make a note in the track and identify methods where this should be updated.